### ISSUE TITLE
Навести порядок в иконках скилов

### ISSUE THREAD

**Issue by hippogamesunity** [hippogamesunity/SpellMasters#56]
Привет! Прошу навести порядок в иконках скилов. Сейчас часть берется из платного пака, часть нарисована, но с пустыми краями. Нужно:
- перенести все используемые иконки из платного пака в Images/ActionIcons (прозрачные области можно пока не дорисовывать)
- все иконки, которые рисуются, должны быть без прозрачных областей, формат PNG, размер 128х128 или 256х256 (предпочтительно). Исходный PSD желательно сохранить и прикопать в ресурсах.

Сцена Battle, путь Canvas1000/Bottom/Skills

![image](https://user-images.githubusercontent.com/7318172/28822599-8724c72e-76c2-11e7-871d-e48d0f683371.png)

**Comment by Ragewortt** [comment]
Т.е. нужно их просто разбить по отдельности с названием скилла и указать новый путь? Плюс закинуть psd.

**Comment by Ragewortt** [comment]
Если с докрашиванием прозрачных областей в 256x256, то иконка вылазит за границы рамки.
[2](https://user-images.githubusercontent.com/22120368/28844659-d2b9c7d2-771e-11e7-9120-e5ce3b509053.jpg)

**Comment by Ragewortt** [comment]
Если меньше, то норм.
![1](https://user-images.githubusercontent.com/22120368/28844658-d2635a0a-771e-11e7-95a0-a6a23fd1b912.jpg)!

**Comment by Ragewortt** [comment]
Если использовать 128x128 с прозрачными краями, то как раз умещается в рамке.
![mortalblow](https://user-images.githubusercontent.com/22120368/28846838-574d20c8-7726-11e7-8dcd-5585ecd2f463.png)

**Comment by hippogamesunity** [comment]
Да, иконку под рамкой нужно уменьшить. Скругленные края и прозрачность не нужны, не будем тащить это их пака.

**Comment by hippogamesunity** [comment]
Размер у RectTransform нужно изменить просто.

**Comment by Ragewortt** [comment]
Сделал разрешение 256х256 и квадратные края.
![3](https://user-images.githubusercontent.com/22120368/28849190-68595f18-772f-11e7-9fdb-ecde41f0d1d0.jpg)
